src/i18n.d.ts

```diff
@@ -3,12 +3,37 @@
 
 import { ComponentType, ReactNode } from 'react'
 
+interface InterpolationOptions {
+    smart_count?: number | { length: number };
+    _?: string;
+    [interpolationKey: string]: any;
+}
+
+interface InterpolationTokenOptions {
```

Comment:
Same as https://github.com/nayaabkhan/react-polyglot/pull/24/files#r329338001 for this.